school_42/d05/ex08_ft_strupcase.py

```diff
@@ -0,0 +1,25 @@
+# Create a function that transforms every letter of every word to uppercase.
+
+def create_uppercase(data_str):
+    upper_str = ""
+    for i in data_str:
+        if ord(i) >= 97 and ord(i) <= 122:
+            symbol = chr(ord(i) - 32)
```

Comment:
move 32 to 'delta' variable. Try to not use some magic numbers in main body of code. It would be better if you declare these magic numbers as variables with readable names in the top of the function/class/file.
you can read this article about usage of magic numbers (in russian =) ) http://www.8bytes.net/2018/04/18/antipattern-1-magicheskoe-chislo/